packages/mobile-app/app/account-select.tsx

```diff
@@ -2,23 +2,41 @@ import { StatusBar } from "expo-status-bar";
 import { Button, StyleSheet, Text, View } from "react-native";
 import { useRouter } from "expo-router";
 import { LinkButton } from "../components/LinkButton";
+import { useQueryClient } from "@tanstack/react-query";
 import { useFacade } from "../data/facades";
 
 export default function AccountSelect() {
   const router = useRouter();
   const facade = useFacade();
+  const qc = useQueryClient();
 
   const getAccountsResult = facade.getAccounts.useQuery(undefined, {
     refetchInterval: 1000,
   });
 
+  const setActiveAccount = facade.setActiveAccount.useMutation({
+    onSuccess: async () => {
+      await qc.invalidateQueries();
```

Comment:
suggestion: We could invalidate queries by default on all mutation changes

Reply:
Yeah I agree. I'll see how we can do this without the TRPC overrides functionality we use in the node app.

Reply:
Figured it out! 🙌